### PR TITLE
Make rsync ssh port show if ssh mode is selected

### PR DESCRIPTION
**Pull request by erincodepirate**
Ticket: #22670

### CODE REVIEW

**Review by pchaussalet — APPROVED** [review]
LGTM

File: ui/inspectors/calendar-task.reel/rsync-args.reel/rsync-args.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ exports.RsyncArgs = AbstractInspector.specialize({
                 this.object.clear();
                 this.object.push({rsync_properties: {}});
                 this.object.__type = this.type;
+                this.object[0].remote_ssh_port = 22;
```
Comment (surajrav) [inline ui/inspectors/calendar-task.reel/rsync-args.reel/rsync-args.js:35]: no need to do this erin, as I have `params.get('remote_ssh_port', 22)` in the middleware already.

Other than that it looks ok. did not get time to test it though

File: ui/inspectors/calendar-task.reel/rsync-args.reel/rsync-args.html [hunk 2]
```diff
@@ -227,8 +236,10 @@
         <div data-montage-id="remoteUser"></div>
         <div data-montage-id="remoteHost"></div>
         <div data-montage-id="remotePath"></div>
-        <div data-montage-id="remotePort"></div>
-        <div data-montage-id="remoteModule"></div>
+        <div data-montage-id="remoteSubstitution">
+            <div data-montage-id="remotePort" data-arg="SSH"></div>
+            <div data-montage-id="remoteModule" data-arg="MODULE"></div>
+        </div>
```
Comment (surajrav) [inline ui/inspectors/calendar-task.reel/rsync-args.reel/rsync-args.html:243]: remote_path is not needed when module is chosen

File: ui/inspectors/calendar-task.reel/rsync-args.reel/rsync-args.js (hunk 1 above)
Comment (pchaussalet) [reply ui/inspectors/calendar-task.reel/rsync-args.reel/rsync-args.js:35]: yes, it needs to be done to let the user know the port he is setting

**Pull request merged**